doc/classes/@GlobalScope.xml

```diff
@@ -2550,6 +2550,155 @@
+		</constant>
+		<constant name="JOY_MODEL_JOYCON_LEFT" value="10" enum="JoyModel">
+			A game controller model that represents the left Nintendo Switch Joycon that is held horizontally.
+			A game controller with this model ID has Face South/Face East/Face West/Face North face buttons, a left stick, SL and SR buttons, Minus/Capture/L/ZL buttons, accelerometer and gyroscope sensors.
+		</constant>
+		<constant name="JOY_MODEL_JOYCON_RIGHT" value="11" enum="JoyModel">
+			A game controller model that represents the right Nintendo Switch Joycon that is held horizontally.
+			A game controller with this model ID has Face South/Face East/Face West/Face North face buttons, a left stick, SL and SR buttons, Plus/Home/R/ZR buttons, accelerometer and gyroscope sensors.
+		</constant>
+		<constant name="JOY_MODEL_JOYCON_PAIR" value="12" enum="JoyModel">
+			A game controller model that represents the pair of Nintendo Switch Joycon controllers, with a similar layout to Nintendo Switch Pro Controller.
+			A game controller with this model ID has B/A/Y/X face buttons, a directional pad, left and right sticks, Minus/Home/Plus buttons, L/R shoulder buttons, ZL/ZR trigger axes (with a value of either 0 or 1), a Capture button, accelerometer and gyroscope sensors (the sensors of the right Joycon controller are being used).
+		</constant>
+		<constant name="JOY_SCHEME_INVALID" value="-1" enum="JoyScheme">
+			An invalid game controller layout scheme.
+		</constant>
+		<constant name="JOY_SCHEME_XBOX" value="0" enum="JoyScheme">
+			A game controller layout scheme that represents the Xbox controller layout.
+			A game controller with this layout scheme has A/B/X/Y face buttons, a directional pad, and it may have left and right sticks, Back/Guide/Start (or View/Xbox/Menu) buttons, LB/RB shoulder buttons and LT/RT trigger axes.
+		</constant>
+		<constant name="JOY_SCHEME_PLAYSTATION" value="1" enum="JoyScheme">
+			A game controller layout scheme that represents the PlayStation controller layout.
+			A game controller with this layout scheme has Cross/Circle/Square/Triangle face buttons,
+			a directional pad, and it may have left and right sticks, Select/PS/Start (or Share/PS/Options) buttons, L1/R1 shoulder buttons, L2/R2 trigger axes and a touchpad in case of PS4 and PS5 controllers.
+		</constant>
+		<constant name="JOY_SCHEME_NINTENDO" value="2" enum="JoyScheme">
+			A game controller layout scheme that represents the Nintendo controller layout.
+			A game controller with this layout scheme has B/A/Y/X face buttons, a directional pad, and it may have left and right sticks, Minus/Home/Plus (or Start/Home/Select) buttons, L/R shoulder buttons, ZL/ZR trigger axes, and possibly a Capture button in case of Nintendo Switch Pro Controller.
```

Comment:
```suggestion
			A game controller with this layout scheme has B/A/Y/X face buttons, a directional pad, and it may have left and right sticks, Minus/Home/Plus (or Start/Home/Select) buttons, L/R shoulder buttons, ZL/ZR trigger axes, and possibly a Capture button in the case of Nintendo Switch Pro Controller.
```